vignettes/decorate-module-output.Rmd

```diff
@@ -12,16 +12,21 @@ vignette: >
 
 ## Introduction
 
-`teal` is a powerful `shiny`-based framework with built-in modules for interactive data analysis. 
-This document outlines the customization options available for modifying the output of `teal` modules. 
-You will learn how to use `teal_transform_module()` to modify and enhance the objects created by `teal::modules()`, 
+`teal` is a powerful `shiny`-based framework with built-in modules for interactive data analysis.
```

Comment:
I'd skip this line at all. This is decorator vignette and everyone should already know what is `teal`. Also "powerful" is a bragging which shouldn't really have a place in the vignette. (this is just my opinion)